tutorial/source/tracking_1d_em.ipynb

```diff
@@ -183,21 +183,23 @@
    "outputs": [],
    "source": [
     "import heapq\n",
-    "def merge_objects(assignment, states_loc, threshold):\n",
+    "def merge_objects(assignment, states_loc, threshold, exists_mask=None):\n",
```

Comment:
Let's split this into two ops: `prune` and `merge`. That will be simpler, clearer, and cheaper (since merging is bottlenecked by the `O(n^2)` computation of pairwise distances, and those only matter for post-pruned objects.